### PR TITLE
GET user/:userSub/reviews endpoint + tests

### PR DESCRIPTION
**Pull request by reillynski**
- completed GET endpoint for user reviews + jest tests
- congito-post-confirmation.test.js (part of add user to dynamodb task) was failing when i ran 'npm test' --> error was "uncaught promise." added a try/catch around docClient call in congito-post-confirmation.js which fixed it so all tests are passing, but i think this is worth looking into because it wasn't failing previously and nothing about it changed (to my knowledge)

### CODE REVIEW

File: backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js [hunk 1]
```diff
@@ -23,7 +23,24 @@ exports.cognitoPostConfirmationHandler = async (event, context, callback) => {
...
+   
+   const response = {
+      statusCode: 200, 
+      body: {
+         results: userResult,
+      },
+   }
+   
+   // callback(null, event);
+   return response;
```
Comment (jpoist97) [inline backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js:45]: I'm a little confused about this change, was this supposed to be included in this pull request?

**Review by jpoist97 — DISMISSED** [review]
Looks good, just a little curious about why you needed to change cognito-post-confirmation

File: backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js (hunk 1 above)
Comment (reillynski) [reply backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js:45]: i am also confused ... i just got this error when i first ran 'npm test' after branching and wasn't sure what caused it bc i didn't change anything in that file

Screen Shot 2022-01-30 at 7 07 54 PM
 
adding a try-catch made the error go away but i don't think the resulting behavior is what we want bc it just caught the same error: 

`TypeError: Cannot read properties of undefined (reading 'promise')`

and the test case wasn't checking for an error being thrown. should def talk abt it more in class tomorrow

**Review by jpoist97 — APPROVED** [review]

**Review by rohithdara — APPROVED** [review]
lgtm !!!

**Review by jpoist97 — APPROVED** [review]

**Pull request merged**